config/config.go

```diff
@@ -50,6 +50,17 @@ const (
 	EnableProfilerKey = "ENABLE_PROFILER"
 	// StatsIntervalKey defines interval for printing basic tdex statistics
 	StatsIntervalKey = "STATS_INTERVAL"
+	// ExperimentalKey defines wether to use an Elements node as the daemon's
+	// block explorer instead of the default Esplora service.
+	ExperimentalKey = "EXPERIMENTAL"
```

Comment:
I think here we could do it a simpler way to decrease number of ENV, I would make `ElementsJsonRpcEndpointKey` and `ExplorerEndpointKey` exclsive, chechink this in the Validate method (ie. throw Fatal error if both are given or neither of two)

Reply:
therefore dropping the experimental, wich could just be documented in the GoDoc comment on top of Elements key

Reply:
consider that atm we set a default value for the Explorer endpoint [here](../blob/master/config/config.go#L65), so technically what you say it's not doable unless we remove that default value.

Reply:
@tiero instead of making them mutually exclusive, I opted for being enough to set a TDEX_ELEMENTS_RPC_ENDPOINT to set the Elements explorer instead of the default Esplora one.

Reply:
Ok so if both set, Elements take precedence? That's ok too (unless we remove the default)